suite-native/module-settings/src/screens/SettingsScreen.tsx

```diff
@@ -25,6 +28,12 @@ export const SettingsScreen = () => {
                 <DeviceSettings />
                 <CoinsSettings />
             </VStack>
+            {/* Only show this button in development mode */}
+            {(!process.env.NODE_ENV || process.env.NODE_ENV === 'development') && (
```

Comment:
simplify to just `process.env.NODE_ENV === 'development'`. First part is not necessary, it worst case it will be undefined so it won't trigger condition anyway.

Reply:
https://github.com/trezor/trezor-suite/pull/5818/commits/74f9395cfeebcdfbc0441d50337f873f3c1b2b4e